lms/djangoapps/support/static/support/jsx/entitlements/components/Main/Main.jsx

```diff
@@ -14,17 +15,45 @@ const Main = props => (
       open={!!props.errorMessage}
     />
     <h2>
-      Entitlement Support Page
+      Student Support: Entitlement
     </h2>
-    <SearchContainer />
-    <EntitlementSupportTableContainer ecommerceUrl={props.ecommerceUrl} />
+    <MainContent
+      isFormOpen={props.isFormOpen}
+      ecommerceUrl={props.ecommerceUrl}
+      openCreationForm={props.openCreationForm}
+    />
   </div>
 );
 
+const MainContent = (props) => {
+  if (props.isFormOpen) {
+    return <EntitlementFormContainer />;
+  } 
+  return (
+    <div>
+      <SearchContainer />
+      <Button
+        className={['btn', 'btn-primary']}
+        label="Create New Entitlement"
```

Comment:
Should we be concerned about translations within this UI?  This may also apply to past PRs.

Reply:
Possibly, I'm not sure how important i18n is for an internal tool though. Who should I check with about that? Additionally, I have no knowledge of how we do i18n for React stuff (though it may just be the same as normal js translations).